src/ContainerLogs.jsx

```diff
@@ -95,47 +95,40 @@ class ContainerLogs extends React.Component {
         }
         this.resize(this.props.width);
 
-        const logsData = {};
-        logsData.name = this.props.containerId;
-
-        varlink.connect(utils.getAddress(this.props.system), this.props.system)
-                .then(connection => {
-                    connection.monitor("io.podman.GetContainerLogs", logsData, this.onStreamMessage)
-                            .then(this.onStreamClose)
-                            .catch(e => {
-                                if (e.error === "ConnectionClosed")
-                                    this.onStreamClose();
-                                else
-                                    this.setState({
-                                        errorMessage: e.message,
-                                        streamer: null,
-                                    });
-                            });
-                    this.setState({
-                        streamer: connection,
-                        errorMessage: "",
-                    });
-                })
+        const connection = rest.connect(client.getAddress(this.props.system), this.props.system);
+        const options = {
+            method: "GET",
+            path: "/v1.12/libpod/containers/" + this.props.containerId + "/logs",
+            body: "",
+            params: {
+                follow: true,
+                stdout: true,
+                stderr: true,
+            },
+        };
+
+        connection.monitor(options, this.onStreamMessage, this.props.system, true)
+                .then(this.onStreamClose)
                 .catch(e => {
                     this.setState({
                         errorMessage: e.message,
                         streamer: null,
                     });
                 });
+        this.setState({
+            streamer: connection,
+            errorMessage: "",
+        });
     }
 
     onStreamMessage(data) {
-        // data = {parameters: {container: [<data_string>]}}
-        // Example <data_string>="2020-01-23T09:34:34.563845178+01:00 stdout F 19\r\n"
-        if (data && data.parameters) {
-            const just_logs = data.parameters.container.map(e => e.split(" ").slice(3)
-                    .join(" "));
+        if (data) {
             if (this.state.loading) {
                 this.state.view.reset();
                 this.state.view._core.cursorHidden = true;
                 this.setState({ loading: false });
             }
-            this.state.view.write(just_logs.join(""));
+            this.state.view.writeln(data.substring(8)); // Line starts with `<0d><01><00><00><00><00><00><00>`
```

Comment:
That *almost* looks like a 64-bit encoded API version, except that it's 13-1, not 12-1 (1.12). What is that? Can that link to some API documentation or so? Or is that a bug in podman?

Reply:
Oh my, yeah, I totally forgot about this thing. I just made it to work and kept it for later investigation but totally forgot about it.
@baude Do you happen to know what is this at beginning of each log line?

Reply:
Would still be nice to at least document this one.

Reply:
What do you mean by "document"? I still have no idea what it is though :)

Reply:
something like "# HACK: Remove garbage at the beginning: <podman issue URL>"?